sdk/endpoints/batch/mnist-nonmlflow.ipynb

```diff
@@ -242,13 +265,13 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## 3.3 Test the endpoint with sample data\n",
+    "## 4.3 Test the endpoint with sample data\n",
     "Using the `MLClient` created earlier, we will get a handle to the endpoint. The endpoint can be invoked using the `invoke` command with the following parameters:\n",
     "- `name` - Name of the endpoint\n",
```

Comment:
`endpoint_name`, instead of `name`? Please double check with SDK spec.

Reply:
https://docs.microsoft.com/en-us/azure/machine-learning/how-to-use-batch-endpoint-sdk-v2 should be updated too. Please add.